src/Zicht/Itertools/lib/Traits/DebugInfoTrait.php

```diff
@@ -14,6 +14,11 @@ trait DebugInfoTrait
      */
     public function __debugInfo(): array // phpcs:ignore Zicht.NamingConventions.Functions.MethodNaming
     {
+        // It seems this method, under unknown circumstances, produces segfaults and exceptions in PHP 8
+        if (PHP_MAJOR_VERSION >= 8) {
+            return [];
+        }
+
```

Comment:
Voelt een beetje als een botte bijl om dit helemaal uit te zetten.  Daarnaast maak je het behoorlijk moeilijk om in dev modus nog dingen te debuggen.  Maar ik snap ook wel dat dit moeilijk is om uit te zoeken.

Zet iig nog even iets in de return, i.e. `return ['__debugInfo has been disabled on PHP 8 because it can cause segfaults'];'.  Anders denkt iedereen dat de lijst leeg is, en dan kun je super lang aan het debuggen zijn...